Access Reviews Graph API overview GDPR note (#16290)
microsoftgraph/microsoft-graph-docs: api-reference/includes/accessreviews-gdpr-overview-note.md

---
author: isabelleatmsft
ms.topic: include
ms.date: 03/15/2021
ms.localizationpriority: medium
---

<!-- markdownlint-disable MD041-->

>[!NOTE]

> >Data of users are included in an access review at the start of the review. Administrators may delete and export the data at any time by using these API methods to support the obligations under GDPR. For general information about GDPR, see the [GDPR section of the Microsoft Trust Center](https://www.microsoft.com/trust-center/privacy/gdpr-overview) and the [GDPR section of the Service Trust portal](https://servicetrust.microsoft.com/ViewPage/GDPRGetStarted).

Reviewer: ```suggestion
>Data about users is included in an access review at the start of the review. Administrators may delete and export the data at any time by using this API to support the obligations under GDPR. For general information about GDPR, see the [GDPR section of the Microsoft Trust Center](https://www.microsoft.com/trust-center/privacy/gdpr-overview) and the [GDPR section of the Service Trust portal](https://servicetrust.microsoft.com/ViewPage/GDPRGetStarted).
```
